src/app/workflow/launch/launch.component.html

```diff
@@ -28,11 +28,16 @@
       Run locally with the Dockstore CLI
       <pre>{{ cli }}</pre>
     </div>
-    <div *ngIf="currentDescriptor === 'cwl'">
-      <div class="well well-sm" tooltip="Commands for launching tools through CWLtool: the CWL reference implementation.
-      cwltool can run a tool directly when all inputs and outputs are available on the local filesystem.">
-        Alternatively, cwltool can run a workflow directly when all inputs and outputs are available on the local filesystem
+    <div *ngIf="(currentDescriptor | lowercase) === 'cwl'">
+      <div class="well well-sm" tooltip="Commands for launching workflows through cwl-runner: the CWL reference implementation.
+      cwl-runner can run a tool directly when all inputs and outputs are available on the local filesystem.">
+        Run locally with cwl-runner
         <pre>{{ cwl }}</pre>
+        <!-- The below is disabled until workflows actually work -->
```

Comment:
Worth pending a test for this on dockstore-webservice

Reply:
The GA4GH production endpoint still works.  It might be a cwltool issue.

Reply:
ok, create a new ticket for us to investigate and bookmark this line then so we follow up

Reply:
Upon further investigation it looks like it does work for workflows.  It appears I was originally testing with a WDL workflow which explains why it didn't work.